tests/installation/yast2_firstboot.pm

```diff
@@ -60,6 +60,7 @@ sub firstboot_timezone {
 sub firstboot_user {
     assert_screen 'local_user';
     enter_userinfo(username => get_var('YAST2_FIRSTBOOT_USERNAME'));
+    wait_screen_change(sub { send_key 'alt-a'; },    7) if is_opensuse;
```

Comment:
Let's put a comment why we need this, so we can see from the code what `atl-a` does. Other than that LGTM.